edx_oauth_client/middleware.py

```diff
@@ -1,24 +1,46 @@
-import re
-import os.path
+import logging
+from urlparse import urljoin, urlparse
 
+import re
```

Comment:
`re` is the default python lib so should be added into the first import block.

Reply:
Was optimized by my IDE and here I totally agree, that optimization was with claims. Fixed the order.